Source/Core/Core/IOS/FS/FileSystemProxy.cpp

```diff
@@ -266,7 +269,8 @@ IPCCommandResult FS::Format(const Handle& handle, const IOCtlRequest& request)
     return GetFSReply(ConvertResult(ResultCode::AccessDenied));
 
   const ResultCode result = m_ios.GetFS()->Format(handle.uid);
-  return GetFSReply(ConvertResult(result));
+  const u64 ticks = result == ResultCode::Success ? SUPERBLOCK_WRITE_TICKS : 0;
+  return GetFSReply(ConvertResult(result), ticks);
```

Comment:
Thoughts on making this an actual function? Checking the result to determine ticks, converting the result and actually returning it seems the same for all of the changes in here (in the superblock commit)